### ISSUE TITLE
[Feature Request] Hide/Truncate Tweets longer than 140 characters

### ISSUE THREAD

**Issue by buckket** [TwidereProject/Twidere-Android#1015]
Twitter recently introduced an increased character limit of 280 characters per Tweet. Not everybody is happy with this change and how it affect ones timeline, thus I propose to include an filter option to Twidere to automatically hide Tweets longer than 140 characters. Additionally another option my be added, which truncates Tweets longer than 140 characters and only shows the full length when specifically selecting the Tweet.

**Comment by jomo** [comment]
You could probably make use of the existing spoiler feature for Mastodon to implement this:

![spoiler1](https://i.imgur.com/RUy4iju.png)

![spoiler2](https://i.imgur.com/p0TKQZB.png)

**Comment by tdemin** [comment]
> Not everybody is happy with this change

Fixing server-side problems with client-side patches? O RLY?

**Comment by AeliusSaionji** [comment]
> Fixing server-side problems with client-side patches?

Yeah, frankly it sounds like @buckket should reevaluate whether they want to continue using Twitter, rather than placing a burden on third party projects with excess code / bloat / wasted development time.

If Twitter has become something you dislike, don't make it someone else's problem.
